arch/arm/src/stm32h7/hardware/stm32h7x3xx_rcc.h

```diff
@@ -812,6 +812,9 @@
 /* RCC Global Control register */
 
 #define RCC_GCR_WW1RSC                  (1 << 0)  /* Bit 0: WWDG1 reset scope control */
+#define RCC_GCR_WW2RSC                  (1 << 1)  /* Bit 1: WWDG2 reset scope control */
```

Comment:
Is this defining reserved bit on other ICs? If so should we use and #ifdef on HAS_XXX to not allow compilations and to prevent errant access to reserved bits?